config/config.yaml.example

```diff
@@ -0,0 +1,2 @@
+ENV_PRODUCT: k3s
+ENV_TFVARS: k3s.tfvars
```

Comment:
same here

Reply:
same as previous comment. sticking with yaml format of `variable: value`

Reply:
same as above

Reply:
Fixed.